CONTRIBUTING.md

```diff
@@ -15,13 +15,13 @@ You will need the following things properly installed on your computer.
 * [Ember CLI](http://www.ember-cli.com/)
 * [PhantomJS 2](http://phantomjs.org/)
 * [Gatekeeper](https://github.com/prose/gatekeeper)
+* [Docker](https://docs.docker.com/install/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
These should not be in the general list of prerequisites, as it is not necessary to use docker.

Reply:
Sure I can figure something out